crates/sui-indexer-alt-schema/migrations/2025-05-19-231614_consistent-pipelines-deletion-reference-table/down.sql

```diff
@@ -0,0 +1,5 @@
+DROP TABLE IF EXISTS obj_info_deletion_reference;
+
+DROP TABLE IF EXISTS coin_balance_buckets_deletion_reference;
+
+DROP TABLE IF EXISTS obj_versions_deletion_reference;
```

Comment:
I'm a little nervous about the size implications of using this technique for `obj_versions`, because it contains so much more historical data (compared to the tables which only store an hour's worth of history).

Supposing we limited `obj_versions` to one or two year's of data but that requires adding deletion references, then do we actually save space? (Or, put another way, how much retention do we need to have in `obj_versions` before it's worth cleaning it up with pruning).